adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -215,6 +216,38 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 			routes = append(routes, routesP...)
 		}
 		return routes, clusters, endpoints, nil
+
+	}
+
+	if mgwSwagger.EndpointType == constants.AwsLambda {
+		for _, resource := range mgwSwagger.GetResources() {
+			amznResourceName := ""
+			for i, operation := range resource.GetOperations() {
+				value := model.ResolveAmznResourceName(operation.GetVendorExtensions())
+
+				if (i != 0) && (amznResourceName != value) {
+					logger.LoggerOasparser.Errorf("ARNs in all the operations of the same resource path should have the same value. | APIID: %v API Title: %v API version: %v", mgwSwagger.GetID(), apiTitle, apiVersion)
+				} else if value == "" {
+					logger.LoggerOasparser.Errorf("ARN cannot be empty. | APIID: %v API Title: %v API version: %v", mgwSwagger.GetID(), apiTitle, apiVersion)
+				} else {
+					amznResourceName = value
+				}
+				resource.SetAmznResourceName(amznResourceName)
+			}
+
+			routesX, err := createRoutes(genRouteCreateParams(&mgwSwagger, resource, vHost, "", awslambdaClusterName, awslambdaClusterName, nil, nil, organizationID, false))
+			if err != nil {
+				logger.LoggerXds.ErrorC(logging.ErrorDetails{
+					Message: fmt.Sprintf("Error while creating routes for AWS Lambda API : %s version : %s. Error: %s",
+						apiTitle, apiVersion, err.Error()),
```

Comment:
If not updated the relevant sheet we can update it considering these codes. 
Please discard this comment if it's already updated.

Reply:
Yes, It's already updated.